### DISCUSSION TITLE
人腦細胞可以玩遊戲，學習速度快過AI

### DISCUSSION THREAD

**Discussion by 109802503gloria** [1101-GS4719-C/Python-Community-GS4719-C#252] [科技新知]
澳洲研究機構成功培養出一群人腦細胞，並教它們學習玩Pong。將這些腦細胞放在高密度微電極陣列上，藉由陣列不同位置的刺激代表球的位置，神經細胞則會放出電流訊號回應並移動球拍，這個學習過程只花了5分鐘，除此之外，腦細胞兼具面對陌生問題的適應性與延伸性。AI雖然學習比較慢，但學習後的表現較佳，研究團隊期望未來能結合機器學習與腦細胞的優點讓AI更進步。

[新聞連結](https://tw.news.yahoo.com/%E6%AF%8D%E9%AB%94-%E6%98%AF%E7%9C%9F%E7%9A%84-%E7%A7%91%E5%AD%B8%E5%AE%B6%E6%95%99%E6%9C%83%E8%BF%B7%E4%BD%A0%E4%BA%BA%E8%85%A6%E7%B4%B0%E8%83%9E%E7%8E%A9%E9%81%8A%E6%88%B2-%E5%AD%B8%E7%BF%92%E9%80%9F%E5%BA%A6%E5%BF%AB%E9%81%8Eai-073331057.html)
原文新聞：[RESEARCHERS TEACH HUMAN BRAIN CELLS IN A DISH TO PLAY “PONG”](https://futurism.com/the-byte/brain-cells-play-pong)

**Comment by zixunn** [comment]
這篇文章介紹當神經元的學習速度與機器學習結合時，可以極大地改進當前的人工智能，這就像是電影情節出現在現實生活中一般，不知是否有朝一日人類會創造出超越人類發展速度的AI機器人。

**Comment by enoch-lightning** [comment]
我們本來就希望人工智能的發展要愈強大愈好，不然就失去發展的意義了。

**Comment by chupochun** [comment]
人腦跟人工智慧的差別就在一個擁有感情，各自都有各自的好處

**Comment by pekoranoinu** [comment]
如果能把腦細胞與機器結合，那不就會變成人造人了嗎

**Comment by xkllkx** [comment]
人腦界面指日可待，說不定之後有刀劍神域(誤)

**Comment by awkward-willy** [comment]
或許透過這樣的技術
能真正的製作結合人類細胞與 AI 的機器人
在許多方面都能有更深入的應用
甚至可能重現電影裡的場面

**Comment by liangyinglly** [comment]
將人腦與機器結合恐怕會涉及倫理問題

**Comment by HaoFish123** [comment]
感謝分享，人腦+AI感覺很可怕。

**Comment by Jimmy911218** [comment]
當人腦細胞加上ai會不會讓ai出現獨立思想值得注意

**Comment by treeann** [comment]
人腦的複雜度目前還是遠超AI

**Comment by sys511613** [comment]
這個技術好酷喔，希望以後可以研究更多人腦的構造去應用在AI上面，或許會出現許多有趣的結果

**Comment by alexthebigpp** [comment]
人腦本來就是一個很先進的電腦，搞不好以後直接拿一整顆腦子來運算。

**Comment by yenwei12138** [comment]
如果能將人腦的學習能力配上AI的大量運算，我們人類就真的要被淘汰了

**Comment by etin2468** [comment]
對於能量消耗觀點來說，人腦感覺比較有效率
